app/services/tokens.js

```diff
@@ -0,0 +1,43 @@
+const { Token } = require('../models');
+const logger = require('../logger');
+const errors = require('../errors');
+
+const createToken = async (userData, token) => {
+  try {
+    logger.info(`tokens-service::createToken::userData::${JSON.stringify(userData)}token::${token}`);
```

Comment:
Do not log tokens

Reply:
Or passwords (if the userData has the field)

Reply:
Ready!